dask/array/tests/test_linalg.py

```diff
@@ -113,7 +113,7 @@ def test_svd_compressed():
     usvt = np.dot(u, np.dot(np.diag(s), vt))
 
     tol = 0.2
-    assert_eq(np.linalg.norm(mat - usvt),
+    assert_eq(np.linalg.norm(usvt),
               np.linalg.norm(mat),
```

Comment:
Yeah what was happening here makes no sense to me. This seems like the best fix without taking this PR in a new direction. Would be nice to have an issue to revisit `test_svd_compressed` for someone to look at later.

Reply:
@mrocklin, do you know more about what this test was trying to do?

Reply:
It was probably trying to compare equality between the original and the recomputed result.  This might be better written as `assert_eq(mat, usvt)` if there is some way to add in a tolerance to `assert_eq` (which there may be, I'm just a little rusty here)

Reply:
> ...if there is some way to add in a tolerance to `assert_eq` (which there may be, I'm just a little rusty here)

Seems like there is as `assert_eq` calls [`allclose`]( https://github.com/dask/dask/blob/20a189cc886f440f461bfc97e5fdf7d2b9cb8a75/dask/array/utils.py#L102 ).

Reply:
> Would be nice to have an issue to revisit `test_svd_compressed` for someone to look at later.

Added as issue ( https://github.com/dask/dask/issues/2695 ).